From da4a1e8a69083f19bb7c4553b04934267d35fdd1 Mon Sep 17 00:00:00 2001
From: Allan Sandfeld Jensen <allan.jensen@digia.com>
Date: Thu, 17 Oct 2013 15:55:24 +0200
Subject: [PATCH 10/20] SVG stroke-dasharray not working

When creating the QPainterPathStroker we forget to also copy the dashPattern property.

Task-number: QTBUG-34063
Change-Id: Idb4d124447e2727c418c8ca5e1de694245f6ba22
---
 Source/WebCore/platform/graphics/qt/GraphicsContextQt.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WebCore/platform/graphics/qt/GraphicsContextQt.cpp b/Source/WebCore/platform/graphics/qt/GraphicsContextQt.cpp
index 3f8aa8d..25d99ea 100644
--- a/Source/WebCore/platform/graphics/qt/GraphicsContextQt.cpp
+++ b/Source/WebCore/platform/graphics/qt/GraphicsContextQt.cpp
@@ -609,6 +609,7 @@ void GraphicsContext::strokePath(const Path& path)
     QPainterPathStroker pathStroker;
     pathStroker.setJoinStyle(pen.joinStyle());
     pathStroker.setDashOffset(pen.dashOffset());
+    pathStroker.setDashPattern(pen.dashPattern());
     pathStroker.setMiterLimit(pen.miterLimit());
     pathStroker.setCapStyle(pen.capStyle());
     pathStroker.setWidth(pen.widthF());
-- 
1.8.5.3

